saturn_client/core.py

```diff
@@ -455,4 +439,10 @@ def _maybe_name(_id):
     """Return message if len of id does not match expectation (32)"""
     if len(_id) == 32:
         return ""
-    return " Maybe you used name rather than id?"
+    return "Maybe you used name rather than id?"
+
+
+def http_error(response: requests.Response, resource_id: str):
```

Comment:
since this is a client library intended for use by customers, I think we should be very very protective of the public API. Can you please change this to `_http_error()`?